article/app/controllers/ArticleController.scala

```diff
@@ -78,7 +78,7 @@ class ArticleController(
   }
 
   private def getJson(article: ArticlePage)(implicit request: RequestHeader): List[(String, Object)] = {
-    val contentFieldsJson = if (request.isGuuiJson) List(
```

Comment:
It looks pretty correct to get rid of guuiJson to just be using forceDCR.

But just to make 100% sure, can you check that .```json?dcr=true``` on an article definitely still works locally?, and produces a different result to ```.json?dcr=true```, which should be equivalent to just ```.json```

Reply:
Yep have verified this!

Reply:
I've verified this :+